chapters/connectors.tex

```diff
@@ -764,7 +764,7 @@ \subsection{Connection Graphs and Their Operators}\label{overconstrained-equatio
   end equalityConstraint;
 end Record;
 \end{lstlisting}
-The array dimension $n$ of \lstinline!residue! shall be a constant \lstinline!Integer! expression that can be evaluated during translation, with $n \geq 0$.
+The array dimension $n$ of \lstinline!residue! shall be an evaluable \lstinline!Integer! expression, with $n \geq 0$.
```

Comment:
This is weaker than the original formulation.  I suggest excluding it from this PR.

Reply:
Reverted

Reply:
OK. Resolving.